src/olympia/addons/tests/test_buttons.py

```diff
@@ -257,7 +257,6 @@ def test_experimental(self):
         b = self.get_button()
         assert not b.featured
         assert b.experimental
-        assert b.button_class == ['caution']
```

Comment:
This will be the default `['download']` again (coming from `InstallButton`) and be tested as well, can you maybe not remove the `assert` but change it to test the `download` class?